site/docs/tutorial/sql_to_substrait.md

```diff
@@ -0,0 +1,245 @@
+# SQL to Substrait tutorial
+
+To provide an introduction to Substrait for those already familiar with SQL,
+this tutorial walks through the basics of Substrait by translating basic SQL
+queries. We'll present the Substrait in JSON form so it is (somewhat) readable
+for newcomers.
+
+Substrait can be a type, an expression, a relation, or a plan. We'll look at them in
+those order, going from simplest to most complex.
+
+## Types and Schemas
+
+<!-- TODO -->
+
+## Expressions
+
+A basic building block of queries are expressions. Let's start simple with the
+SQL expression:
+
+```sql
+a > 10
+```
+
+An expression can be one of several things. `a` itself will be translated to
+a field reference:
+
+```json
+{
+  "selection": {
+    "directReference": {
+      "structField": {
+        "field": 1
```

Comment:
Might be worth noting that Substrait refers to fields numerically based on some kind of schema / field order (depends on the operator).  In SQL field references are always named references.

Reply:
Good point. I'll make sure to call that out. I'll need to look a little deeper to, since the natural question is "relative to *which* schema?"